src/CustomReport/Schema/CustomReportColumnConfiguration.php

```diff
@@ -0,0 +1,77 @@
+<?php
+declare(strict_types=1);
+
+/**
+ * Pimcore
+ *
+ * This source file is available under two different licenses:
+ * - GNU General Public License version 3 (GPLv3)
+ * - Pimcore Commercial License (PCL)
+ * Full copyright and license information is available in
+ * LICENSE.md which is distributed with this source code.
+ *
+ *  @copyright  Copyright (c) Pimcore GmbH (http://www.pimcore.org)
+ *  @license    http://www.pimcore.org/license     GPLv3 and PCL
+ */
+
+namespace Pimcore\Bundle\StudioBackendBundle\CustomReport\Schema;
+
+use OpenApi\Attributes\Property;
+use OpenApi\Attributes\Schema;
+
+/**
+ * @internal
+ */
+#[Schema(
+    title: 'Custom Report Column Configuration',
+    type: 'object',
+)]
+final readonly class CustomReportColumnConfiguration
+{
+    public function __construct(
+        #[Property(description: 'Name', type: 'string', example: 'attributesAvailable')]
+        private string $name,
+        #[Property(description: 'Display name of column', type: 'bool', example: true)]
+        private string $display,
+        #[Property(description: 'Whether the column should be included in exports', type: 'bool', example: true)]
+        private string $export,
+        #[Property(description: 'Order', type: 'bool', example: true)]
+        private string $order,
+        #[Property(description: 'Label/display name of column', type: 'string', example: 'Attributes')]
+        private string $label,
+        #[Property(description: 'Id', type: 'string', example: 'extModel401-3')]
```

Comment:
not sure if we still need that example. Ext would be a extjs thing. Maybe something else might be better, but I am sure this will come up once FE is ready to implement that.

Reply:
True, I will change the example as soon as we know how column ids in the new ui will look.